src/app/containers/FrontPageMain/index.test.jsx

```diff
@@ -57,14 +57,17 @@ describe('FrontPageMain', () => {
     fetch.mockResponse(JSON.stringify(pidginMostReadData));
   });
 
-  // TODO: This is temporarily due to most read issues and will be added back in ASAP
-  // describe('snapshots', () => {
-  //   it('should render a pidgin frontpage correctly', async () => {
-  //     await matchSnapshotAsync(
-  //       <FrontPageMainWithContext frontPageData={frontPageData} />,
-  //     );
-  //   });
-  // });
+  describe('snapshots', () => {
+    it('should render a pidgin frontpage correctly', async () => {
+      const { container } = render(
+        <FrontPageMainWithContext frontPageData={frontPageData} />,
+      );
+
+      await waitForElement(() => container.querySelector('#Most-Read'));
```

Comment:
This PR looks good. 
It'd be good to add a comment here in the code that we are waiting for Most Read since it only renders after data is fetched separately.

Reply:
Added a comment